livekit/_utils.py

```diff
@@ -22,3 +24,85 @@ async def get(self) -> T:
             await self._event.wait()
         self._event.clear()
         return self._queue.popleft()
+
+
+class BroadcastQueue(Generic[T]):
```

Comment:
Better naming for BroadcastQueue & BroadcastChannel.
Feels confusing rn

Reply:
agreed it's a bit confusing.

also a bit confusing that `subscribe` returns a queue for clients to manage.

Reply:
if the main difference between _Queue and _Channel is thread safety? why not call it ThreadsafeBroadcastQueue?